README.md

```diff
@@ -88,17 +91,26 @@ Color: whitesmoke (#F5F5F5)
 - `in progress`: Someone is working on it.  
 - `sprint ready`: Ready to estimate.  
 
+**Close labels**  
+Color: white (#FFFFFF)  
+Close the issue when assigning one of these.  
+- `wont fix`: This will not be fixed for an arbitrary reason (e.g. a crash in a part of the App that will be removed soon).  
+Before closing, provide the reason as comment if possible.  
+- `duplicate`: Title says it all. Always keep the issue with the most information.  
+Before closing, add comment ```closed as duplicate of #<issue-that-is-kept>``` to remove the need of migrating the information to the kept issue.  
```

Comment:
I think we don't need labels for those, just adding a won't fix or duplicate comment when closing it will gives the information we need about why it was closed. Why do we need a tag for them?